api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1207 @@
+            $ref: '#/definitions/Problem'
+    get:
+      tags:
+        - topic-api
+        - draft
+      description: |
+        Starts a stream delivery for the specified partitions of the given topic.
+
+        The event stream is formated as a sequence of `EventStreamBatch`es separated by `\n`. Each
+        `EventStreamBatch` constains a chunk of Events and a `Cursor` pointing to the end of the chunk (i.e.
+        last delivered Event).
+        If streaming for several distinct partitions, each one is a independent `EventStreamBatch`.
+
+        The initialization of a stream can be parameterized in terms of size of each chunk, timeout for
+        flushing each chunk, total amount of delivered Events and total time for the duration of the stream.
+
+        Nakadi will keep a streaming connection open even if there are no events to be delivered. In this
+        case the timeout for the flushing of each chunk will still apply and the `EventStreamBatch` will
+        contain only the Cursor pointing to the same offset. This can be treated as a keep-alive control for
+        some load balancers.
+
+        The tracking of the current position in the partitions and of which partitions is being read is in
+        the responsibility of the client.
+        No commits are needed.
+      parameters:
+        - name: name
+          in: path
+          description: Topic where to get events from
+          type: string
+          required: true
+        - name: X-nakadi-cursors
+          in: header
+          description: Array of cursors indicating the partitions to read from and the starting point on
+            each. Assumes the offset on each cursor is not inclusive (i.e., first delivered Event is the
+            **first one after** the one pointed to in the cursor).
+          required: false
+          type: array
+          items:
+            type: string
+            format: '#/definitions/Cursor'
+        - name: batch_limit
+          in: query
+          description: |
+            Maximum number of `Event`s in each chunk (and therefore per partition) of the stream.
+
+            * If 0 or unspecified will buffer Events indefinitely and flush on reaching of
+            `batch_flush_timeout`.
+          type: integer
+          format: int32
+          required: false
+          default: 1
+        - name: stream_limit
+          in: query
+          description: |
+            Maximum number of `Event`s in this stream (over all partitions being streamed in this
+            connection).
+
+            * If 0 or undefined, will stream batches indefinately.
+
+            * Stream initialization will fail if `stream_limit` is lower than `batch_limit`.
+          type: integer
+          format: int32
+          required: false
+          default: 0
+        - name: batch_flush_timeout
+          in: query
+          description: |
+            Maximum time in seconds to wait for the flushing of each chunk (per partition).
```

Comment:
@igoryankov hm... I though that in such a situation you probably will not need batching at all and would like to get everything as fast as possible. 

If we see that somebody wants to created batches every half a second for example, we can extend this interface to accept number with a floating point and not int32 value.

Somehow seconds are easier to think about usually.

Please fill free to insist on milliseconds if you have some nice examples